doc/modules/decomposition.rst

```diff
@@ -799,6 +799,10 @@ Maximizing ELBO is equivalent to minimizing the Kullback-Leibler(KL) divergence
 between :math:`q(z,\theta,\beta)` and the true posterior
 :math:`p(z, \theta, \beta |w, \alpha, \eta)`.
 
+The :attr:`n_topics` of :class:`LatentDirichletAllocation` can be used to set the number of topics, :math:`K`.
```

Comment:
Just curious, feel free to ignore - Why :attr:`n_topics` and not simply ``n_topics``?